### PR TITLE
Add Rust binding and ScaleIO exporter

### PR DESCRIPTION
**Pull request by syepes**

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -87,7 +87,8 @@ hosted outside of the Prometheus GitHub organization.
    * [WebDriver exporter](https://github.com/mattbostock/webdriver_exporter)
    * [Xen exporter](https://github.com/lovoo/xenstats_exporter)
    * [Zookeeper exporter](https://github.com/carlpett/zookeeper_exporter)
-
+   * [ScaleIO exporter](https://github.com/syepes/sio2prom)
```
Comment (grobie) [inline content/docs/instrumenting/exporters.md]: Please sort alphabetically.

File: content/docs/instrumenting/exporters.md [hunk 2]
```diff
@@ -121,3 +122,4 @@ practices.
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
    * Java/JVM: [Hystrix metrics publisher](https://github.com/soundcloud/prometheus-hystrix)
+   * Rust: [rust-prometheus](https://github.com/pingcap/rust-prometheus/)
```
Comment (grobie) [inline content/docs/instrumenting/exporters.md]: That's a client library and already documented on https://prometheus.io/docs/instrumenting/clientlibs/

**Comment by syepes** [comment]
@grobie Did not see that one ;-)

File: content/docs/instrumenting/exporters.md (hunk 1 above)
Comment (grobie) [inline content/docs/instrumenting/exporters.md]: @syepes

**Comment by syepes** [comment]
@grobie done ;-)

**Pull request merged**